config/deployment.yml

```diff
@@ -36,16 +36,12 @@ apiVersion: v1
 kind: Service
 metadata:
   name: python-example-service
-  annotations:
-    external-dns.alpha.kubernetes.io/hostname: "doctl-action.do-api.dev"
-    service.beta.kubernetes.io/do-loadbalancer-certificate-id: "37f354b1-0043-4bef-84b9-4666390026f3"
-    service.beta.kubernetes.io/do-loadbalancer-redirect-http-to-https: "true"
 spec:
   type: LoadBalancer
   ports:
-    - name: https
+    - name: http
       protocol: TCP
-      port: 443
-      targetPort: 5000
+      port: 80
+      targetPort: 80
```

Comment:
This app listens on port 80, so we need to update the `targetPort`.

https://github.com/sharmita3/sample-python/blob/main/server.py#L16

Since we removed the SSL configuration above, we need to change the `port` to 80 as well.